contracts/airdrop/AirdropRewarder.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {ERC20Upgradeable} from "@openzeppelin/contracts-upgradeable/token/ERC20/ERC20Upgradeable.sol";
+import {MerkleProof} from "@openzeppelin/contracts/utils/cryptography/MerkleProof.sol";
+import {Initializable} from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+contract AirdropRewarder is Initializable, OwnableUpgradeable {
+    using SafeERC20 for ERC20Upgradeable;
+    address public safeAddress;
+
+    mapping(address => bytes32) private merkleRoots;
```

Comment:
better to keep merkleroot a constant bytes32; since we are only airdropping one token. ZERO

Reply:
Done